### ISSUE TITLE
Make hosts in inventory/group_vars/all.yml dynamic

### ISSUE THREAD

**Issue by siwon** [TOSIT-IO/tdp-getting-started#93]
Hi and first of all : thanks for your work !

I'm trying to apply this on vm without vagrant, and I think this part of `inventory/group_vars/all.yml` file should by dynamic based on inventory.
In fact, IPs (which are dynamics) and hostname are wrong in my case.

https://github.com/TOSIT-IO/tdp-getting-started/blob/930956873a8129ea10d570658d068217a1d57bbc/inventory/group_vars/all.yml#L3-L11

**Comment by leopaul36** [comment]
Hello,

These variables are used to generate a `Vagrantfile` (which you won't need) and the Ansible `inventory/hosts` file. The generation happens at the `ansible-playbook generate-node-deployment-config.yml` step.

Replace the IPs with whatever your VMs IPs berfore running the command are and you should be good to go. You might also want to manually update the `ansible_ssh_private_key_file` key in the generated inventory to connect to your VMs.

**Comment by siwon** [comment]
I've already managed the `inventory/hosts` file but didn't planned to edit the `groups_vars` files (until now).

Indeed, I still have to edit these hosts values because they are used to generate the `/etc/hosts` files on servers (in the `deploy-vms.yml` playbook).

To do so, I wrote a preconfig playbook but my way of doing it didn't suit me. Even a small change in the `groups_vars`  could make my pre-playbook not working anymore ...

Anyway, I should have mentionned that this issue is more a question/feature request than a real issue (my bad).

As I have a workaround for my usecase, I let you decide if this issue is still relevant.
